__docs__/components/all-variants.tsx

```diff
@@ -141,7 +141,6 @@ export function AllVariants(props: Props) {
                                         >
                                             {children({
                                                 props: {
-                                                    "aria-label": ariaLabel,
```

Comment:
This was always injecting the `aria-label` prop into the child component. Removing this because it can be invalid to set the aria-label depending on the element. We can explicitly set the aria-label of a component when needed by using the `name` property that we provide to the `children` render function

This addresses the `Ensure ARIA attributes are not prohibited for an element's role` warning in Storybook for the Badge stories